.azure-pipelines.yml

```
@@ -44,7 +44,7 @@ jobs:
       python -m pip install --upgrade pip
       pip install -r .pip-2033.txt
       pip install pytest-cov wheel
-      pip install .[dev,doc,test,louvain,leiden,magic,scvi,harmony,scrublet,scanorama]
+      pip install .[dev,doc,test,louvain,leiden,magic,harmony,scrublet,scanorama]
```

```suggestion
      pip install .[dev,doc,test,louvain,leiden,magic,harmony,scrublet,scanorama,skmisc]
```

I think this should fix the azure builds